examples/mugen/retrieval/README.md

```diff
@@ -0,0 +1,41 @@
+# MUGEN Retrieval
+
+This directory contains model components for MUGEN's video-text retrieval model, a tutorial notebook for the model usage (better viewed on [nbviewer](https://nbviewer.org/github/facebookresearch/multimodal/blob/main/examples/mugen/retrieval/evaluation.ipynb)), and training and evaluation scripts.
+
+## Model
+MUGEN's video-text retrieval model follows from [VideoCLIP](https://arxiv.org/abs/2109.14084), a contrastive model for video and text.
+
+The name "VideoCLIP" refers to its similarities to OpenAI's [CLIP](https://arxiv.org/abs/2103.00020), which was originally proposed for zero-shot learning of image classification tasks by “drawing cues” from text data with the corresponding visual concepts. Unlike various predecessor models based on supervised learning, CLIP does not have to be trained on the task-specific datasets or fine-tuned with a task-specific head. The model learns a joint embedding space for both image and text data and optimizes a scaled cosine similarity function between the image and text embedding vectors. The loss function is the sum of the normalized cosine similarities for every pair of image-and-text samples. Each embedding is trained with a modality specific transformer encoder, e.g., a multi-head transformer for text, vision transformer (ViT) for image.
+
+The VideoCLIP model follows the CLIP architecture but replaces the image encoder with a video encoder. VideoCLIP's video encoder is backed by S3D, a video classification model, and the text encoder is backed by DistilBERT, a lightweight transformer for language modeling.
```

Comment:
Can we use the full name for S3D and hyper link it?
Hyper link DistilBERT as well.

Reply:
added links to the s3d and distilbert arxiv pages